docs/docusaurus/docs/cloud/quickstarts/airflow_quickstart.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Airflow'
+title: 'Quickstart for GX Cloud and Airflow'
+id: airflow_quickstart
+description: Connect GX Cloud to an Airflow Orchestrator.
+---
+
+In this quickstart, you'll learn how to use GX Cloud with Apache Airflow. Apache Airflow is an orchestration tool that allows you to schedule and monitor your data pipelines. For more information about Apache Airflow, see the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/index.html).
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud).
+
+- You have the [Astro CLI](https://docs.astronomer.io/astro/cli/overview) installed.
+
+- You have [connected GX Cloud to a Data Asset on a Data Source](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset).
+
+- You have [created an Expectation Suite](/docs/cloud/expectation_suites/manage_expectation_suites.md) and [added Expectations](/docs/cloud/expectations/manage_expectations#create-an-expectation).
+
+- You have [added a Checkpoint to your Expectation](/docs/cloud/checkpoints/manage_checkpoints#add-a-checkpoint).
+
+
+## Create a local Airflow project and set dependencies
+
+1. Open a terminal, navigate to the directory where you want to create your Airflow project, and then run the following code:
+
+    ```bash title="Terminal input"
+    mkdir gx-cloud-airflow && cd gx-cloud-airflow
+    astro dev init
+    ```
+    After running the code, a new directory is created, you're taken to that directory, and a new Airflow project is initialized.
+
+2. Browse to the directory where you created your Airflow project, open the `requirements.txt` file, and then add the following text as a new line: 
+
+    ```
+    airflow-provider-great-expectations==0.2.7
+    ```
+
+    This text adds the GX Airflow Provider to the Airflow project.
+    
+3. Save your changes and close the `requirements.txt` file.
+
+4. Open the `packages.txt` file and add the following text as a new line:
+
+    ```
+    libgeos-c1v5
```

Comment:
What is libgeos-c1v5 used for?

Reply:
This was an apparent dependency that we had, but I just tried this all again without this line and it seems to work. I'm inclined to leave this in for now until I'm able to get someone else to verify.